pkg/cluster/sync.go

```diff
@@ -110,6 +111,24 @@ func (c *Cluster) Sync(newSpec *acidv1.Postgresql) error {
 		}
 	}
 
+	// remove unused PVCs in case deleting them during scale down failed; see Update()
+	// the last pvc stays until the cluster is explicitly deleted as opposed to being scaled down to 0 pods
+	if c.OpConfig.EnableUnusedPVCDeletion && c.getNumberOfInstances(&c.Spec) > 0 {
+
+		// XXX that also deletes PVC of pods shut down before this change is deployed
```

Comment:
Triple xXx?

Reply:
It marks the code that requires special attention, like `FIXME`, `HACK`, and other similar markers